modules/kegg_completeness.nf

```diff
@@ -0,0 +1,42 @@
+process KEGG_COMPLETENESS {
+
+    label 'process_light'
+
+    publishDir(
+        path: "${params.outdir}",
+        saveAs: {
+            filename -> {
+                def cluster_prefix = cluster.substring(0, cluster.length() - 2);
+                return "species_catalogue/${cluster_prefix}/${cluster}/genome/${filename}";
+            }
+        },
+        mode: "copy",
+        failOnError: true
+    )
+
+    container 'https://depot.galaxyproject.org/singularity/kegg-pathways-completeness:1.0.5--pyhdfd78af_0'
+
+    input:
+    tuple val(cluster), file(eggnog_annotation_tsvs)
+
+    output:
+    tuple val(cluster), path("*_kegg_pathways.tsv"), emit: pathways
+
+    script:
+    """
+    # Prepare input
+    prep_pathway_completeness_input.py -e ${eggnog_annotation_tsvs} -o ${cluster}_kos.txt
```

Comment:
@KateSakharova should we consider adding to support for eggnog in the kegg_pathway tool? As as a convenience thingy